installation/proc-config-dynamic-plugins-rhdh-operator.adoc

```diff
@@ -0,0 +1,70 @@
+[id="proc-config-dynamic-plugins-rhdh-operator_{context}"]
+= Configuring dynamic plugins with the {product} Operator
+
+You can store the configuration for dynamic plugins in a ConfigMap object that the Custom Resource can reference.
+
+.Example using the GitHub dynamic plugin
+In {ocp-short}, you can use the following example as a base template to create a ConfigMap named `dynamic-plugins-rhdh`:
+
+[source, yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: dynamic-plugins-rhdh
+data:
+  dynamic-plugins.yaml: |
+    includes:
+      - dynamic-plugins.default.yaml
+    plugins:
+      - package: './dynamic-plugins/dist/backstage-plugin-catalog-backend-module-github-dynamic'
+        disabled: false
+        pluginConfig: {}
+----
+
+[NOTE]
+--
+If the `pluginConfig` field references environment variables, you must define the variables in your `secrets-rhdh` secret.
+--
```

Comment:
I think it would be better to include this in the procedure if possible, maybe shown as an example after step 2 or 3

Reply:
I agree. I think the placement of this can be improved, but this is existing content that was moved, but not created or changed as part of this PR. Let's... create a separate Jira ticket for this.